citylistmetadata/tests/parsing.test.ts

```diff
@@ -31,3 +31,27 @@ test('Parse single subject', () => {
         },
     })
 })
+
+test('Normalize city name case', () => {
+    expect(normalizeTitleCase('PEC POD SNĚŽKOU')).toBe('Pec pod Sněžkou')
+    expect(normalizeTitleCase('KLÁŠTEREC NAD OHŘÍ')).toBe('Klášterec nad Ohří')
+    expect(normalizeTitleCase('ROKYTNICE V ORLICKÝCH HORÁCH')).toBe('Rokytnice v Orlických horách')
+    expect(normalizeTitleCase('KOSTELEC NAD ČERNÝMI LESY')).toBe('Kostelec nad Černými lesy')
+    expect(normalizeTitleCase('ŽĎÁR NAD SÁZAVOU')).toBe('Žďár nad Sázavou')
+    expect(normalizeTitleCase('FRYŠAVA POD ŽÁKOVOU HOROU')).toBe('Fryšava pod Žákovou horou')
+    expect(normalizeTitleCase('PRAHA-ÚJEZD')).toBe('Praha-Újezd')
```

Comment:
U víceslovných bychom mohli oddělit pomlčkou místo spojovníkem, vypadá to pak lépe :)

> Spojovník se píše k vyznačení těsného spojení v místních jménech a v názvech správních oblastí, např. Brno-Slatina, Frýdek-Místek, Garmisch-Partenkirchen, Praha-Hlubočepy, Bohumín-sever, Ostrava-město, Praha-západ. Pokud je alespoň jedna ze složek víceslovná, lze z důvodu přehlednosti nahradit spojovník pomlčkou oddělenou mezerami, tedy např. Praha 6 – Ruzyně, Liberec VII – Horní Růžodol, Karlovy Vary – Drahovice, Brno – Královo Pole (viz Pomlčka).
>
> Upozorňujeme, že náhrada spojovníku pomlčkou oddělenou mezerami není vhodná v případech, kdy by mohlo dojít k textové nejednotě (např. v seznamech adres) či k významové nejednoznačnosti (např. při zápisu cest, dopravních spojení, vedení silnic, stezek nebo tras výletů). V takových případech doporučujeme ponechat spojovník, např. železniční trať Ústí nad Labem-Střekov – Děčín hl. n., výluka v úseku Praha-Vysočany – Praha-Horní Počernice – Lysá nad Labem, trasa výletu Praha-Velká Chuchle – Černošice-Mokropsy – Řevnice – Karlštejn apod
>